iree/test/micro_benchmarks/run_benchmark_on_android.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+MLIR_FILE_PATH=${1}
+ARGS=${@:2}
+
+adb shell 'mkdir /data/local/tmp'
+MLIR_FILE_NAME=$(basename $MLIR_FILE_PATH)
+TARGET_VM_FILE=/tmp/$MLIR_FILE_NAME.fbvm
```

Comment:
vmfb :)

Reply:
The plan Is this should work on any linux-machine with local android phone attached =), 
I don't mind deleting but was to make it easy to reproduce the same numbers I have: 

had for this is users need to are:
1) Build an android release build
2) Define IREE_RELEASE_DIR and run the script.